breathecode/activity/urls/v2.py

```diff
@@ -16,5 +16,6 @@
     path('me/activity/<str:activity_id>', V2MeActivityView.as_view(), name='me_activity_id'),
     path('academy/activity', V2AcademyActivityView.as_view(), name='academy_activity'),
     path('academy/activity/<str:activity_id>', V2AcademyActivityView.as_view(), name='academy_activity_id'),
+    path('report', V2AcademyActivityReportView.as_view(), name='report'),
```

Comment:
It looks like `activity/:user_id/report` if this endpoint were based in the activity dataset, this will be a good option